advanced/am-pattern-2/templates/am/pub-store-tm/instance-2/wso2am-pattern-2-instance-1-am-volume-claim.yaml

```diff
@@ -23,14 +23,7 @@ spec:
   resources:
```

Comment:
@Chathuru Let us rename this file as it contains multiple PVCs. Also, the filename refers to `instance-1`.

Reply:
Renamed https://github.com/wso2/kubernetes-apim/pull/365/commits/656646538e8cc1b8d8fbc51cb3d206a1c58101e5